ci/meta.yml

```diff
@@ -41,12 +41,13 @@ meta:
           args:
             - -exc
             - |
-              cp ../terraform-config/terraform.tf .
+              cp ../../../terraform-config/terraform.tf .
+              cp ../../../terraform-config/terraform.tfvars .
               terraform workspace show
               terraform init
               terraform plan -out terraform.plan
               terraform apply -auto-approve terraform.plan
-          dir: dataworks-metrics-infrastructure
+          dir: dataworks-metrics-infrastructure/terraform/deploy
```

Comment:
I think it might be better to leave this as `dataworks-metrics-infrastructure` and put `cd terraform/deploy` in the above args?

Reply:
Discussed, this pattern is used in aws-concourse. If needs be this can be modified later on, but seems fine for now.